providers/src/airflow/providers/amazon/aws/sensors/rds.py

```diff
@@ -104,18 +104,17 @@ class RdsExportTaskExistenceSensor(RdsBaseSensor):
 
     :param export_task_identifier: A unique identifier for the snapshot export task.
     :param target_statuses: Target status of export task
+    :param error_statuses: Target error status of export task to fail the sensor
     """
 
-    template_fields: Sequence[str] = (
-        "export_task_identifier",
-        "target_statuses",
-    )
+    template_fields: Sequence[str] = ("export_task_identifier", "target_statuses", "error_statuses")
```

Comment:
to keep the same format

```suggestion
    template_fields: Sequence[str] = (
        "export_task_identifier",
        "target_statuses",
        "error_statuses",
    )
```

Reply:
ruff format is changing it.